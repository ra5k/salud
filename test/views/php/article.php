<?php $script->extend("partials/layout.php", function ($base) { ?>


    <?php $base->block("header", function () { ?>
        <nav>HEADER</nav>
    <?php }) // block:main ?>


    <?php $base->block("main", function () { ?>

        <div class="main">
            <?= $this->call('partials/time.php') ?>
        </div>

    <?php }) // block:main ?>


    <?php $base->block("footer", 'FOOTER STROM STRING'); ?>

<?php }); // extends

