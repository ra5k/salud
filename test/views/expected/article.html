<html>
    <head>
        <meta charset="UTF-8" />
        <title>The Title</title>
    </head>
    <body>
        <header>
                    <nav>HEADER</nav>
    
        </header>
        
        <main>
        
        <div class="main">
            <div class="article">ARTICLE Epoch: 123456 </div>
        </div>

    
        </main>

        <footer>
            FOOTER STROM STRING
        </footer>
    </body>
</html>
